File: storydoc-backend/src/main/java/io/storydoc/server/ui/app/ScreenShotCollectionDTO.java
package io.storydoc.server.ui.app;

import io.storydoc.server.storydoc.app.dto.StoryDocSummaryDTO;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class ScreenShotCollectionDTO {

    private String name;

    private StoryDocSummaryDTO storyDocSummary;

    private List<ScreenShotDTO> screenShots;

}

File: storydoc-frontend/src/app/api/models/screen-shot-collection-dto.ts
/* tslint:disable */
/* eslint-disable */
import { ScreenShotDto } from './screen-shot-dto';
import { ScreenshotCollectionCoordinate } from './screenshot-collection-coordinate';
import { StoryDocSummaryDto } from './story-doc-summary-dto';
export interface ScreenShotCollectionDto {
  coordinate?: ScreenshotCollectionCoordinate;
  name?: string;
  screenShots?: Array<ScreenShotDto>;
  storyDocSummary?: StoryDocSummaryDto;
}
